lib/vdsm/storage/sp.py

```diff
@@ -1205,6 +1210,11 @@ def activateSD(self, sdUUID):
         if dom.getDomainClass() == sd.DATA_DOMAIN:
             self._convertDomain(dom)
 
+        # If running as SPM, check that autoactivation is disabled for all VGs
+        # before activating the SD.
```

Comment:
AFAIR `SD` is "engine" language, while `Pool` is used in VDSM :)
Nir, please express your opinion.